config/locales/models/user/en.yml

```diff
@@ -0,0 +1,73 @@
+# Structure needed for Rails i18n active record messages management
+# Example file
+en:
```

Comment:
Great! I'll do that and also add the corresponding documentation on how to use this translation structure the right way!